goserver/session/store.go

```diff
@@ -0,0 +1,64 @@
+package session
+
+import (
+	"encoding/json"
+	"errors"
+	"time"
+)
+
+// ErrNotFound is the error returned when something is not found.
+var ErrNotFound = errors.New("not found")
+
+// Store provides functions to store/retrieve keyed binary data.
+type Store interface {
+	// Store stores a key-value pair with an expiration time.
+	Store(key string, val []byte, expiresAt time.Time) error
```

Comment:
Can't we just encode the expiresAt time as part of the []byte to simplify this interface?

Reply:
Done.